crates/apub/src/objects/post.rs

```diff
@@ -34,66 +27,117 @@ use lemmy_db_schema::{
   },
 };
 use lemmy_utils::{
-  location_info,
   request::fetch_iframely_and_pictrs_data,
-  utils::{check_slurs, convert_datetime, remove_slurs},
+  utils::{check_slurs, convert_datetime, markdown_to_html, remove_slurs},
   LemmyError,
 };
 use lemmy_websocket::LemmyContext;
 use url::Url;
 
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Page {
+  #[serde(rename = "@context")]
+  context: OneOrMany<AnyBase>,
+  r#type: PageType,
+  pub(crate) id: Url,
+  pub(crate) attributed_to: Url,
+  to: [Url; 2],
+  name: String,
+  content: Option<String>,
+  media_type: MediaTypeHtml,
```

Comment:
The posts are always markdown tho right.

Reply:
The `content` field is HTML (for compat with other software), while the `source` field has markdown.